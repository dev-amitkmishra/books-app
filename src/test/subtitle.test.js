import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Subtitle from '../components/atoms/subtitle';

configure({ adapter: new Adapter() });

describe('Subtitle', () => {
    it('should be defined', () => {
        expect(Subtitle).toBeDefined();
    });
    it('should render correctly', () => {
        const tree = shallow(
            <Subtitle name='subtitle test' />
        );
        expect(tree).toMatchSnapshot();
    });
});