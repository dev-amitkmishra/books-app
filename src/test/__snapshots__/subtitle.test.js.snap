// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subtitle should render correctly 1`] = `ShallowWrapper {}`;
